packages/klighd-core/src/proxy-view/proxy-view-options.ts

```diff
@@ -0,0 +1,525 @@
+    static readonly CHOICE_OFF: string = "Off";
+    static readonly CHOICE_STRAIGHT_EDGE_ROUTING: string = "Straight-Edge-Routing";
+    static readonly CHOICE_ALONG_BORDER_ROUTING: string = "Along-Border-Routing";
+    static readonly DEFAULT: string = ProxyViewEnableEdgeProxies.CHOICE_OFF;
+    static readonly CHOICES: string[] = [
+        ProxyViewEnableEdgeProxies.CHOICE_OFF,
+        ProxyViewEnableEdgeProxies.CHOICE_STRAIGHT_EDGE_ROUTING,
+        ProxyViewEnableEdgeProxies.CHOICE_ALONG_BORDER_ROUTING
+    ];
+    readonly id: string = ProxyViewEnableEdgeProxies.ID;
+    readonly name: string = ProxyViewEnableEdgeProxies.NAME;
+    readonly type: TransformationOptionType = TransformationOptionType.CHOICE;
+    readonly initialValue: string = ProxyViewEnableEdgeProxies.DEFAULT;
+    readonly description: string = ProxyViewEnableEdgeProxies.DESCRIPTION;
+    readonly renderCategory: string = ProxyViewCategory.ID;
+    readonly values: string[] = ProxyViewEnableEdgeProxies.CHOICES;
+    currentValue = ProxyViewEnableEdgeProxies.DEFAULT;
+}
+
+/**
+ * Whether segment proxies should be created.
+ * Segment proxies connect off-screen segments of an edge.
+ */
+export class ProxyViewEnableSegmentProxies implements RenderOption {
+    static readonly ID: string = "proxy-view-enable-segment-proxies";
+    static readonly NAME: string = "Enable Segment Proxies";
+    static readonly DESCRIPTION: string =
+        "Whether segment proxies should be created. " +
+        "Segment proxies connect off-screen segments of an edge.";
+    static readonly DEFAULT: boolean = true;
+    readonly id: string = ProxyViewEnableSegmentProxies.ID;
+    readonly name: string = ProxyViewEnableSegmentProxies.NAME;
+    readonly type: TransformationOptionType = TransformationOptionType.CHECK;
+    readonly initialValue: boolean = ProxyViewEnableSegmentProxies.DEFAULT;
+    readonly description: string = ProxyViewEnableSegmentProxies.DESCRIPTION;
+    readonly renderCategory: string = ProxyViewCategory.ID;
+    currentValue = ProxyViewEnableSegmentProxies.DEFAULT;
+}
+
+/** Whether proxies should be interactable. */
+export class ProxyViewInteractiveProxies implements RenderOption {
+    static readonly ID: string = "proxy-view-interactive-proxies";
+    static readonly NAME: string = "Interactive Proxies";
+    static readonly DESCRIPTION: string = "Whether proxies should be interactable. Clicking on a proxy hops to its off-screen counterpart.";
+    static readonly DEFAULT: boolean = true;
+    readonly id: string = ProxyViewInteractiveProxies.ID;
+    readonly name: string = ProxyViewInteractiveProxies.NAME;
+    readonly type: TransformationOptionType = TransformationOptionType.CHECK;
+    readonly initialValue: boolean = ProxyViewInteractiveProxies.DEFAULT;
+    readonly description: string = ProxyViewInteractiveProxies.DESCRIPTION;
+    readonly renderCategory: string = ProxyViewCategory.ID;
+    currentValue = ProxyViewInteractiveProxies.DEFAULT;
+}
```

Comment:
I think this should be a debug option, the interactive features should always be turned on.